app/containers/AccountInitializer/selectors.js

```diff
@@ -0,0 +1,11 @@
+import { createSelector } from 'reselect';
+import { initialState } from './reducer';
+
+const selectAccountInitializerDomain = state =>
+  state.get('accountInitializer', initialState);
+
+const makeSelectAccountInitializer = () =>
```

Comment:
Let's separate select functions for each state value. That way it is easier to understand which values are available in the state. Also account value will be used separately from loading and error.